src/tools/driver/help_text.dart

```diff
@@ -33,6 +35,16 @@ Supported things:
   poi
```

Comment:
I'll do that when it's hooked into the driver good, and no - log cat is not a user friendly name. Rename to follow on subsequent PR, with integration to the primary driver.